tests/test_service.py

```diff
@@ -131,7 +131,7 @@ def test_properties():
     ret = json.loads(response.body)
 
     # check the data
-    assert(len(ret) == 34 and ret['id_prefixes'][0] == 'CHEBI' and ret['definition_uri'] == 'biolink:ChemicalSubstance' and 'molecularly interacts with' in ret['slots'])
+    assert(len(ret) == 45 and ret['id_prefixes'][0] == 'CHEBI' and ret['class_uri'] == 'biolink:ChemicalSubstance' and 'in taxon' in ret['slots'])
```

Comment:
This change doesn't make any sense to me.  molecularly interacts with should still be one of the slots I think.

I don't understand why "in taxon" would be a slot for chemical, though it seems to be in the live version as well.

Reply:
OK, looks like "in taxon" is there because it's added into a parent of chemical.  There's no problem with that, I guess. But molecularly intereacts with should still be there.

Reply:
so should i raise an issue in biolink model?

Reply:
I guess - what do the slots look for chemical substance?

Reply:
I already raised an issue about taxon